spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -165,6 +166,9 @@
       given_there_is_a_scheduled_step_by_step_page
       and_I_visit_the_publish_or_delete_page
       then_I_see_an_unschedule_button
+      and_I_click_the_unschedule_button
```

Comment:
Suggest renaming to `when_I_click_the_unschedule_button`

Reply:
how about `when_I_unschedule_publishing`? This seems to be the way of naming methods with `click_on` action

Reply:
👍